Controller/DbControllers/RequestController.cs

```diff
@@ -39,6 +39,7 @@ public virtual List<Request> GetArtistRequests()
         }
 
 
+
```

Comment:
enters weghalen

Reply:
uberhaüpt bij alle files die je commit hebt

Reply:
komt door mergen, dan gebeurt dit

Reply:
nee, zo werkt dat niet. als dat zo is dan ben je de enige die dat heeft

Reply:
Maar haal je ze even weg? :)